CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
   </summary>
 </details>
 
+## 0.6.0 (February 26, 2019)
+- GA added
```

Comment:
```suggestion
- Google Analytics added
```

Let's try to avoid using too many acronyms in our changelogs -- explicit > implicit.